examples/default-config/main.tf

```
@@ -3,5 +3,5 @@ provider "lacework" {}
 provider "aws" {}
 
 module "aws_config" {
-  source = "../../"
+  source = "lacework/config/aws"
```

The same comment, we need this to stay as-is:
```suggestion
  source = "../.."
```